### PR TITLE
feat(observe): observe and mlflow implementation

### PR DESCRIPTION
**Pull request by GALLLASMILAN**
I implemented the observability to the starter repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,23 @@
...
 4. Start the example `npm run start` (it runs the ./src/agent.ts file).
+
+## Using observability
+
+> [!NOTE]
+>
+> Docker distribution with support for compose is required, the following are supported:
+>
+> - [Docker](https://www.docker.com/)
+> - [Rancher](https://www.rancher.com/) - macOS users may want to use VZ instead of QEMU
```
Comment (Tomas2D) [inline README.md:17]: Mention `Podman`

File: tsconfig.json [hunk 2]
```diff
@@ -18,5 +18,6 @@
     "strict": true,
     "skipLibCheck": true,
     "strictNullChecks": true
-  }
+  },
+  "exclude": ["node_modules", "dist", "eslint.config.js", "prettier.config.js"]
 }
```
Comment (Tomas2D) [inline tsconfig.json:23]: Why these changes are needed?

**Review by Tomas2D — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 2 above)
Comment (GALLLASMILAN) [reply tsconfig.json:23]: Does not work with `eslint.config.js` and ohter files.

**Review by Tomas2D — APPROVED** [review]
